packages/matplotlib/src/html5_canvas_backend.py

```diff
@@ -56,8 +69,11 @@ def draw(self):
                 return
             ctx = canvas.getContext('2d')
             renderer = RendererHTMLCanvas(ctx, width, height,
-                                          dpi=self.figure.dpi)
+                                          self.figure.dpi, self)
             self.figure.draw(renderer)
+            self.draw_counter += 1
+            if self.draw_counter == 3:
+                window.plot_updated = True
```

Comment:
This number `3` is probably specific to a particular test.  Instead, lets just put the counter in `window.draw_counter` and have the test code (in `test/test_matplotlib.py`) wait on that.

Reply:
Okay, so rather than waiting on `window.plot_updated`, we now wait on `window.draw_counter` to become `3` right?